docs/compass/04-01-installation.md

```diff
@@ -459,7 +458,7 @@ global:
 #      tlsCrt: ${TLS_CERT}
 #      tlsKey: ${TLS_KEY}
 ```
-Start the Database installation, can be skipped if it was installed during [Ory installation with local persistence](#in-cluster-database):
+Start the Database installation, can be skipped if it was installed during [Ory Installation with Local Persistence](#in-cluster-database):
```

Comment:
Start the Database installation, can be skipped if it was installed during the Ory installation with local persistence. See [Ory Installation with Local Persistence](#in-cluster-database):